core/src/Microsoft.Identity.Core.csproj

```diff
@@ -57,6 +57,7 @@
     <None Remove="Cache\MsalAccessTokenCacheKey.cs" />
     <None Remove="Cache\MsalIdTokenCacheKey.cs" />
     <None Remove="Cache\MsalRefreshTokenCacheKey.cs" />
+    <None Remove="Platforms\net45\UserPassCred.cs" />
```

Comment:
><None Remove="Platforms\net45\UserPassCred.cs" /> [](start = 4, length = 49)

should be moved to the net45 item group or the code should be moved to a common place if truly usable across the other clients (did you validate if it's possible to use this flow from Xamarin/UWP/others?

Reply:
Again, I don't know why Visual Studio added that line for me. I thought it was significant, but I tried simply removing it just now, and nothing blows up. So I guess I would just remove it in my next commit. Let me know if you have different opinion. #ConsiderItDone?